### ISSUE TITLE
Bug add child component

### ISSUE THREAD

**Issue by Nikon5070** [nuxt-community/storybook#35]
### Version
@nuxtjs/storybook: 
nuxt: 

component `Wrapper.vue`
```<template>
  <div class="wrapper">
    Wrapper - rounded = {{ rounded }}
    <WrapperItem />
    <slot />
  </div>
</template>

<script>
export default {
  name: 'Wrapper',

  props: {
    rounded: {
      type: Boolean,
      default: false,
    },
  },
}
</script>
```

story `Wrapper.stories.js`
```export default {
  title: 'Wrapper',
}

export const asAComponent = () => '<Wrapper :rounded="true">rounded</Wrapper>'
```

We look in the browser console
![image](https://user-images.githubusercontent.com/15028461/89126200-3f42ba80-d527-11ea-87db-eb2dede29368.png)

**Comment by farnabaz** [comment]
Hi,
This will fix in https://github.com/nuxt-community/storybook/pull/31

**Comment by farnabaz** [comment]
Fixed in [v1.1.0](https://github.com/nuxt-community/storybook/releases/tag/v1.1.0)
